Fix(parse cs): Remove '>' char from path fields

arulths opened the pull request:
- added a new list that includes any fields that may have paths in them
  - note that this list is not exhaustive, it only includes the path fields already defined in `cryosparc2.py`
- strategy: remove the first occurrence of the "`>`" char in each path using `str.replace()`

fixes https://github.com/asarnow/pyem/issues/102

asarnow commented:
Cool, thanks! This is almost ready to be merged, can you instead add a PATH_FIELDS list to `star.Relion` containing the field constants (like e.g. `COORDS3D` or any of those lists), and then instead of `Series.apply()` with a lambda it's probably faster to use `Series.str.replace` (I think `df[field].str.replace(">", "", 1)`).

arulths commented:
Thanks @asarnow, I've made the modifications. I didn't end up removing the lambda as I tested this section with 1.2M rows and two path fields, here were my findings:

### with lambda:
test 1
```
Replaced > with empty string in rlnMicrographName
Replaced > with empty string in ucsfImagePath
Time taken to replace > with empty string: 0.3381047248840332
```
test 2
```
Replaced > with empty string in rlnMicrographName
Replaced > with empty string in ucsfImagePath
Time taken to replace > with empty string: 0.3361241817474365
```

### with Series.str.replace:
test 1
```
Replaced > with empty string in rlnMicrographName
Replaced > with empty string in ucsfImagePath
Time taken to replace > with empty string: 0.37374305725097656
```
test 2:
```
Replaced > with empty string in rlnMicrographName
Replaced > with empty string in ucsfImagePath
Time taken to replace > with empty string: 0.39196300506591797
```

Its not a big amount but it is slower, but let me know if you'd still like me to use Series.str.replace anyway

asarnow commented:
Nice, thank you! I'm going to merge the PR now.

Keeping .apply() is fine, I guess Series.str.replace is doing some extra stuff that's a bit slower. There's a pandas issue about it [here](https://github.com/pandas-dev/pandas/issues/35864).
